src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.WindowsSettings/Classes/WindowsSetting.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using Microsoft.PowerToys.Run.Plugin.WindowsSettings.Enumerations;
+
+namespace Microsoft.PowerToys.Run.Plugin.WindowsSettings.Classes
+{
+    /// <summary>
+    /// A windows setting
+    /// </summary>
+    internal class WindowsSetting : IWindowsSetting
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="WindowsSetting"/> class.
+        /// </summary>
+        public WindowsSetting()
+        {
+            Name = string.Empty;
+            Area = string.Empty;
+            Command = string.Empty;
+            Type = WindowsSettingsType.SettingsApp;
```

Comment:
Lets instantiate with simple string value. (See comment on enumeration too.)
```suggestion
            Type = string.Empty;
```

Reply:
You can't do that, Type is an enum, not a string.

Reply:
Oooops. But what I mean is removing/not using the enum interface. Because I don't think we need it. The enum brings complexity.

The only use case here would be to prevent incorrect settings types. And I think we don't need to do it. If we have incorrect values they can't get translated based on resx data. And if we do translation for settings type like on the other values, we can prevent this by catching the translation error.

Reply:
changed, the enumerable is no longer in